packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -174,7 +181,9 @@ const AccountTransferForm = observer(
 
         const is_mf_status_pending = mf_account_status === MT5_ACCOUNT_STATUS.PENDING;
         const is_mf_status_need_verification = mf_account_status === MT5_ACCOUNT_STATUS.NEEDS_VERIFICATION;
-        const is_mf_status_pending_or_needs_verification = is_mf_status_pending || is_mf_status_need_verification;
+        const is_mf_status_verification_failed = mf_account_status === MT5_ACCOUNT_STATUS.FAILED;
+        const is_mf_status_pending_or_needs_verification =
+            is_mf_status_pending || is_mf_status_need_verification || is_mf_status_verification_failed;
```

Comment:
@farrah-deriv For is_mf_status_pending, is_mf_status_need_verification, is_mf_status_verification_failed,is_mf_status_pending_or_needs_verification can we avoid checking against MT5_ACCOUNT_STATUS object. MT5_ACCOUNT_STATUS is for MT5 accounts, here we are checking deriv MF account status. Maybe we can just check  const is_mf_status_pending = mf_account_status === 'pending' for now to avoid confusion